### PR TITLE
feat(perf-issues): hardcode org to always allow creation for demo

### PR DESCRIPTION
**Pull request by DominikB2014**
This is a temp change, just to hardcode a specific org for the demo.
We want it to always create an issue for this org, despite it being PHP.

### CODE REVIEW

File: src/sentry/utils/performance_issues/performance_detection.py [hunk 1]
```diff
@@ -353,15 +353,13 @@ def _detect_performance_problems(
...
-    for span in spans:
-        detector.visit_span(span)
+def run_detector_on_data(detector, data, project=None):
+    if detector.is_event_eligible(data, project):
+        spans = data.get("spans", [])
+        for span in spans:
+            detector.visit_span(span)
 
-    detector.on_complete()
+        detector.on_complete()
```
Comment (mjq-sentry) [inline src/sentry/utils/performance_issues/performance_detection.py:362]: Aside from the addition of `project`, is the control flow change here just cosmetic? I think it is but just want to be sure. (FWIW I find it easier for review (and safer!) to minimize refactoring lines when you're making logic changes at the same time - basically, when possible make either logic changes or refactors but not both in a single PR 😄)

Comment (DominikB2014) [reply src/sentry/utils/performance_issues/performance_detection.py:362]: Yes, sorry that's mb. I originally moved out `is_event_eligible` outside of `run_detector_on_data`, but moved it back. I'll revert that!

Comment (DominikB2014) [reply src/sentry/utils/performance_issues/performance_detection.py]: @mjq-sentry should be good now!

File: src/sentry/utils/performance_issues/performance_detection.py [hunk 2]
```diff
@@ -353,8 +353,8 @@ def _detect_performance_problems(
     return list(unique_problems)
 
 
-def run_detector_on_data(detector, data):
-    if not detector.is_event_eligible(data):
+def run_detector_on_data(detector, data, project=None):
```
Comment (DominikB2014) [inline src/sentry/utils/performance_issues/performance_detection.py:356]: defaulting to None so refactoring all test cases isn't needed.

**Review by mjq-sentry — APPROVED** [review]
🚢 🎤

**Pull request merged**